### PR TITLE
Music genres and transfer to Siberia option were added.

### PR DESCRIPTION
**Pull request by DeccemberGirl**
Music genres were added to tracks. Transfer to Siberia option was added too.

### CODE REVIEW

File: Managers/TrackManager.cs [hunk 1]
```diff
@@ -30,6 +31,10 @@ public ValidationResult CreateTrack(TrackDTO trackDTO)
             {
                 return new ValidationResult("The track link must not be empty.");
             }
+            if (trackDTO.GenreId == (int)GenreEnum.SmthRussian)
+            {
+                return new ValidationResult("Fuck off. To listen to russian music go to Siberia.");
```
Comment (DatAnton) [inline Managers/TrackManager.cs:36]: please be more tolerant=)

File: Managers/TrackManager.cs [hunk 2]
```diff
@@ -39,6 +44,11 @@ public ValidationResult CreateTrack(TrackDTO trackDTO)
 
         public ValidationResult ChangeTrack(int trackId, TrackDTO trackDTO)
         {
+            if (trackDTO.GenreId == (int)GenreEnum.SmthRussian)
+            {
+                return new ValidationResult("Check the genre you want to set and try again or fuck off to Siberia.");
```
Comment (DatAnton) [inline Managers/TrackManager.cs:49]: please

**Review by DatAnton — APPROVED** [review]
good job

File: Managers/TrackManager.cs (hunk 1 above)
Comment (DeccemberGirl) [reply Managers/TrackManager.cs:36]: Won`t fix

File: Managers/TrackManager.cs (hunk 2 above)
Comment (DeccemberGirl) [reply Managers/TrackManager.cs:49]: Won`t.

**Comment by DeccemberGirl** [comment]
> good job

Thanks

File: Managers/TrackManager.cs (hunk 1 above)
Comment (DatAnton) [reply Managers/TrackManager.cs:36]: remove this genre

Comment (DeccemberGirl) [reply Managers/TrackManager.cs:36]: Remove russians

Comment (DeccemberGirl) [reply Managers/TrackManager.cs:36]: Ladno, let`s not fuck russians. Just sent `em to Siberia. I`ll rewrite

Comment (DeccemberGirl) [reply Managers/TrackManager.cs]: Ladno

File: Managers/TrackManager.cs [hunk 3]
```diff
@@ -30,6 +31,10 @@ public ValidationResult CreateTrack(TrackDTO trackDTO)
             {
                 return new ValidationResult("The track link must not be empty.");
             }
+            if (trackDTO.GenreId == (int)GenreEnum.SmthRussian)
+            {
+                return new ValidationResult("To listen to russian music go to Siberia.");
```
Comment (megablochesk) [inline Managers/TrackManager.cs:36]: о це так фіча

Comment (DeccemberGirl) [reply Managers/TrackManager.cs:36]: Thanks for approve, it is a pleasure to work with you!

**Pull request merged**